src/app/common/EntityStatus.vue

```diff
@@ -46,15 +46,15 @@ const entityStatusClassNames = computed(() => {
 }
 
 .entity-status.is-online {
-  color: var(--green-400);
+  color: var(--green-500);
 }
 
 .entity-status.is-offline {
   color: var(--red-600);
 }
 
 .entity-status.is-degraded {
-  color: var(--custom-orange);
+  color: #ff8040;
```

Comment:
doh this is a shame isn't it. None of these are ideal but:

1. Did you see theres a `yellow-400`/`yellow-500` in kongponents that is reasonably close to this, not sure maybe its a bit off
2. Is it worth making an `orange-400` just to try and keep getting rid of hex's everywhere

None of the above are ideal for different reasons, but what do you think?

Hmm I just spotted this also:

![yellow](https://user-images.githubusercontent.com/554604/207054569-418f6e56-8abd-484f-abca-c1d6f2ffb8c6.png)

After seeing that, I think `yellow-500` might be the best thing instead of having a custom orange, whether with a custom variable or hex? wdyt?